script/4_PowerSimulationsDynamics_examples/01_omib.jl

```diff
@@ -81,29 +81,32 @@ PSID.execute!(
 
 # ## Exploring the solution
 
+# First, we need to load the simulation results into memory
+results = read_results(sim)
+
 # `PowerSimulationsDynamics` has two functions to obtain different
 # states of the solution:
 #  - `get_state_series(sim, ("generator-102-1", :δ))`: can be used to obtain the solution as
 # a tuple of time and the required state. In this case, we are obtaining the rotor angle `:δ`
 # of the generator named `"generator-102-1"`.
 
-angle = get_state_series(sim, ("generator-102-1", :δ));
-Plots.plot(angle, xlabel = "time", ylabel = "rotor angle [rad]", label = "rotor angle")
+angle = get_state_series(results, ("generator-102-1", :δ));
+plot(angle, xlabel = "time", ylabel = "rotor angle [rad]", label = "rotor angle")
 
 # - `get_voltagemag_series(sim, 102)`: can be used to obtain the voltage magnitude as a
```

Comment:
It should say `get_voltage_magnitude_series(results, 102)` instead of `get_voltagemag_series(sim, 102)`